changelogs/fragments/1830-valmod_docmissingtype_batch1.yml

```
@@ -0,0 +1,11 @@
+bugfixes:
+  - kibana_plugin - ``state`` parameter choices must use ``list()`` in python3
+  - elasticsearch_plugin - ``state`` parameter choices must use ``list()`` in python3
+  - riak - parameters ``wait_for_handoffs`` and ``wait_for_ring`` are ``int`` but the default value was ``False``
+  - logstash_plugin - wrapped ``dict.keys()`` with ``list`` for use in ``choices`` setting
+minor_changes:
+  - runit - removed unused code
+  - runit - passing command as ``list`` instead of ``str`` to ``run_command()``
```

```suggestion
  - runit - removed unused code, and passing command as ``list`` instead of ``str`` to ``run_command()`` (https://github.com/ansible-collections/community.general/pull/1830).
```
Please move this into the `bugfixes:` section.